vllm/worker/neuron_model_runner.py

```diff
@@ -57,20 +56,13 @@ class NeuronModelRunner(ModelRunnerBase[ModelInputForNeuron]):
 
     def __init__(
         self,
-        model_config: ModelConfig,
-        parallel_config: ParallelConfig,
-        scheduler_config: SchedulerConfig,
-        device_config: DeviceConfig,
+        vllm_config: ModelConfig,
```

Comment:
Is this meant to be `EngineConfig`?

Reply:
nice catch!

Reply:
fixed in [02242c7](https://github.com/vllm-project/vllm/pull/9938/commits/02242c7d5e0fa794b4f05ca97b4dc1e93356f250)